monolithic/monolithic-hexagonal/monolithic-hexagonal-loader/pom.xml

```diff
@@ -9,10 +9,13 @@
         <version>0.0.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.epam.microservices.monolithic</groupId>
```

Comment:
why? what's the effect of `groupId` removal?

Reply:
It is the same as in the parent `pom.xml` so the declaration here is redundant.